src/screens/TestCases/Modal/CustomFilterModal.jsx

```diff
@@ -0,0 +1,420 @@
+    },
+    {
+      value: 'is between',
+      label: 'Between'
+    },
+    {
+      value: 'is not between',
+      label: 'Is Not Between'
+    }
+  ];
+
+  // <option value="equals">Equals</option>
+  // <option value="does not equal">
+  //   Does Not Equals
+  // </option>
+  // <option value="begins with">Begins With</option>
+  // <option value="does not begin with">Does Not Begin With</option>
+
+  // <option value="ends with">End with</option>
+  // <option value="does not end with">Does Not End with</option>
+
+  // <option value="contains"> Contains</option>
+  // <option value="does not contain">
+  //   Does Not Contain
+  // </option>
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (containerRef.current) {
+        const { top, left } = containerRef.current.getBoundingClientRect();
+        if (window.innerWidth < 600) {
+          containerRef.current.style.top = '10%';
+          containerRef.current.style.left = '5%';
+          containerRef.current.style.right = '5%';
+          containerRef.current.style.bottom = 'auto';
+        } else {
+          containerRef.current.style.top = `${top}px`;
+          containerRef.current.style.left = `${left}px`;
+        }
+      }
+    };
+
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const handleShow = () => setShowModal(true);
+  const handleCloseModal = () => setShowModal(false);
+  const dispatch = useDispatch();
+
+  // const handleSubmit = () => {
+  //   const filterPayload = {
+  //     filter_testcase_data: [
+  //       {
+  //         column: filterColumnId,
+  //         searchText: filterText,
+  //         filter: filterType
+  //       }
+  //     ]
+  //   };
+
+  //   console.log('filterPayload', filterPayload);
+
+  //   // dispatch(
+  //   //   getDraftTestCaseList({
+  //   //     filter_testcase_data:filterPayload ,
+  //   //     limit: paginationData.rowPerPage,
+  //   //     page: paginationData.currentPage
+  //   //     // Spread the filter payload into the request payload
+  //   //   })
+  //   // );
+
+  //   dispatch(
+  //     getDraftTestCaseList({
+  //       limit: paginationData.rowPerPage,
+  //       page: paginationData.currentPage,
+  //       filter_testcase_data: [
+  //         {
+  //           column: filterColumnId,
+  //           searchText: filterText,
+  //           filter: filterType
+  //         }
+  //       ]
+  //     })
+  //   );
+
+  //   handleCloseModal();
+  // };
+
+  if (!show) return null;
+
+  return (
+    <div
+      ref={containerRef}
+      className="custom-container"
+      style={{ top: position.top, left: position.left }}
+    >
+      <div>
+        <b className="fs-6 text-custom">{columnName}</b>
+        <hr className="my-2" />
+
+        <p className="Sort mb-2">Sort</p>
+        <span className="d-flex flex-column">
+          <p
+            className="fs-6 mb-2 pointer"
```

Comment:
why 2 class with same purpose? 
1.Pointer 
2.Cusrsor-Pointer

User "cp" which globally avaible class for same purpose

Reply:
resolved